build.gradle.kts

```diff
@@ -1,7 +1,11 @@
 plugins {
-    id("org.jetbrains.dokka") version "1.9.0"
+    id("org.jetbrains.dokka")
 }
 
 repositories {
     mavenCentral()
 }
+
+tasks.named<Wrapper>("wrapper") {
```

Comment:
Could you add a comment here to explain what this code does?

Reply:
Will do